README.md

```diff
@@ -122,6 +122,45 @@ spec:
 
 **Note:** `spec.type` can take values in ['website','library','service'] but to render GitLab Entity, Catalog must be of type `service`
 
+## Old/New Gitlab Versions
+
+If you have an old Gitlab version, or a new one, we allow you to extend the Gitlab Client as follow:
+
+`packages/app/src/api.ts`
+
+```ts
+import { GitlabCIApiRef } from '@immobiliarelabs/backstage-plugin-gitlab';
+
+export const apis: AnyApiFactory[] = [
+    createApiFactory({
+        api: GitlabCIApiRef,
+        deps: { configApi: configApiRef, discoveryApi: discoveryApiRef },
+        factory: ({ configApi, discoveryApi }) =>
+            new CustomGitlabCIClient({
+                discoveryApi,
+                baseUrl: configApi.getOptionalString('gitlab.baseUrl'),
+            }),
+    }),
+];
+```
+
+`myCustomClient.ts`
```

Comment:
where i put this file? :)

Reply:
This can be put where you want.

Reply:
The precedenting file is a backstager file this is not.

Reply:
I added a general path.